src/Storages/MergeTree/ReplicatedMergeTreeSink.cpp

```diff
@@ -800,7 +801,7 @@ std::pair<std::vector<String>, bool> ReplicatedMergeTreeSinkImpl<async_insert>::
 
             /// When we attach existing parts it's okay to be in read-only mode
```

Comment:
We should probably update this comment too.

Reply:
done